Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+# Build stage
+FROM rust:latest as builder
+WORKDIR /usr/src/hxckr-core
+COPY . .
+RUN apt-get update && apt-get install -y libpq-dev
+RUN cargo install diesel_cli --no-default-features --features postgres
+RUN cargo build --release
+
+# Runtime stage
+FROM debian:bookworm-slim
+RUN apt-get update && apt-get install -y libpq-dev && rm -rf /var/lib/apt/lists/*
+COPY --from=builder /usr/src/hxckr-core/target/release/hxckr-core /usr/local/bin/hxckr-core
+COPY --from=builder /usr/local/cargo/bin/diesel /usr/local/bin/diesel
+COPY --from=builder /usr/src/hxckr-core/migrations /migrations
+COPY entrypoint.dev.sh /entrypoint.sh
```

Comment:
We are copying `entrypoint.dev.sh` to /entrypoint.sh, I don't think `entrypoint.dev.sh` exist. Am I missing it?

Reply:
Fixed in [chore: renamed entrypoint file](https://github.com/hxckr-org/hxckr-core/pull/24/commits/7f72b4c4e67e42dd2ad98a019683cc5d5cc73a3d)